android/src/amazon/AndroidManifest.xml

```diff
@@ -1,13 +1,12 @@
-
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
           package="com.dooboolab.RNIap">
-<application>
+<application android:exported="true">
```

Comment:
@SergiOnGit Could you kindly explain how this fixes the compatibility issue? As far as I know this just allows another app to start the activity.

Reply:
@hyochan Sorry, looks like I wrote another library's PR title here.

Reply:
@hyochan Can you accept this PR? This is really important.

Reply:
I am sorry for the late review. @SergiOnGit Could you kindly explain how this is important in `amazon` IAP?

Reply:
@hyochan This is blocking build on Android, don't if should be exported true or false, but it's blocking. #1571 

```bash
> Task :react-native-iap:processAmazonReleaseManifest FAILED
/Users/boristacyniak/Projects/Mobile/Monstera/node_modules/react-native-iap/android/src/main/AndroidManifest.xml Error:
        android:exported needs to be explicitly specified for <receiver>. Apps targeting Android 12 and higher are required to specify an explicit value for `android:exported` when the corresponding component has an intent filter defined. See https://developer.android.com/guide/topics/manifest/activity-element#exported for details.

```

Reply:
> @SergiOnGit Could you kindly explain how this fixes the compatibility issue? As far as I know this just allows another app to start the activity.

I don't know native code, I just know that without this app doesn't build. Tell you the truth it is very bad that this kind of good module have this bug for this long. This doesn't happen on windows only, I have tested on mac too and it's the same.